cves/CVE-2013-4315.yml

```diff
@@ -49,7 +49,11 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  In versions 1.4.x and 1.5.x of Django, an oversight in ssi template tags granted
```

Comment:
I don't know what an oversight means? can you explain that a little more.

Reply:
This use of oversight implies that the ssi tag was not even thought of to cause an accidental increase in authorization to gain access to the root directories.